### ISSUE TITLE
[Bug]: Incompatibility with FancyMenu

### ISSUE THREAD

**Issue by criscky** [Yesssssman/epicfightmod#1273]
### Have you checked if a similar issue is already reported by someone else?

- [X] I checked there are no similar issues have been reported.

### Have you read the support policy?

- [X] I read it and I accept the policy.

### Are you using the latest Epic Fight and recommended Forge version?

- [X] I checked I'm using latest Epic Fight and recommended Forge version.

### Is this issue related to mod incompatibility?

- [X] This is a mod compatibility issue and I'm aware of the problem.

### The mod

https://www.curseforge.com/minecraft/mc-mods/fancymenu-forge

### Minecraft Version

1.16.5

### What happened?

when i use FancyMenu and try to render the player in the main menu the log keeps spamming the following error, causing extreme lag: 

```[19:00:54] [Render thread/ERROR] [ne.mi.ev.EventBus/EVENTBUS]: Exception caught during firing event: null
	Index: 1
	Listeners:
		0: NORMAL
		1: ASM: class yesman.epicfight.client.events.engine.RenderEngine$Events renderLivingEvent(Lnet/minecraftforge/client/event/RenderLivingEvent$Pre;)V
java.lang.NullPointerException
	at net.minecraft.world.World.func_82736_K(World.java:919)
	at yesman.epicfight.client.events.engine.RenderEngine$Events.renderLivingEvent(RenderEngine.java:430)
	at net.minecraftforge.eventbus.ASMEventHandler_100_Events_renderLivingEvent_Pre.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
	at net.minecraft.client.renderer.entity.LivingRenderer.func_225623_a_(LivingRenderer.java:51)
	at de.keksuccino.fancymenu.menu.fancy.item.playerentity.PlayerEntityCustomizationItem$MenuPlayerRenderer.render(PlayerEntityCustomizationItem.java:454)
	at de.keksuccino.fancymenu.menu.fancy.item.playerentity.PlayerEntityCustomizationItem.renderEntityStatic(PlayerEntityCustomizationItem.java:320)
	at de.keksuccino.fancymenu.menu.fancy.item.playerentity.PlayerEntityCustomizationItem.lambda$renderPlayerEntity$0(PlayerEntityCustomizationItem.java:294)
	at com.mojang.blaze3d.systems.RenderSystem.runAsFancy(SourceFile:1439)
	at de.keksuccino.fancymenu.menu.fancy.item.playerentity.PlayerEntityCustomizationItem.renderPlayerEntity(PlayerEntityCustomizationItem.java:293)
	at de.keksuccino.fancymenu.menu.fancy.item.playerentity.PlayerEntityCustomizationItem.render(PlayerEntityCustomizationItem.java:240)
	at de.keksuccino.fancymenu.menu.fancy.menuhandler.MenuHandlerBase.onRenderPost(MenuHandlerBase.java:1265)
	at net.minecraftforge.eventbus.ASMEventHandler_81_MenuHandlerBase_onRenderPost_Post.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
	at net.minecraftforge.client.ForgeHooksClient.drawScreenInternal(ForgeHooksClient.java:364)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:356)
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:492)
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:977)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607)
	at net.minecraft.client.main.Main.main(Main.java:184)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

used mods:
EpicFight-16.6.4
fancymenu_forge_2.7.2_MC_1.16.2-1.16.5
konkrete_forge_1.5.0_MC_1.16.2-1.16.5

used config for FancyMenu: 
[fancymenu.zip](https://github.com/Yesssssman/epicfightmod/files/10494385/fancymenu.zip)

Suggestion: possibly check if the level isnt null before https://github.com/Yesssssman/epicfightmod/blob/2c1d963891810184a3b6c0c3f0a794b4d08129c4/src/main/java/yesman/epicfight/client/events/engine/RenderEngine.java#L430

i know you might not support this version anymore, but this could be happening in other versions as well, and if you could fix it for 1.16.5 me and [@adam9899](https://github.com/adam9899) would be very grateful, thanks for the attention

**Comment by MetalKnight56** [comment]
We'll look into this, can you please check if this issue persists on 1.18.2?

**Comment by criscky** [comment]
indeed it persists, you can use the same config i sended in a 1.18.2 instance and it will do the same
tested with the following mods:
EpicFight-18.3.7
fancymenu_forge_2.13.3_MC_1.18.2
konkrete_forge_1.6.0_MC_1.18-1.18.2

**Comment by MetalKnight56** [comment]
Thanks, we'll see if we can fix this

**Comment by MetalKnight56** [comment]
Closing issue as fixed on 18.5.0.1 - available only for Closed Beta Access users
